lib/LoginStrategy.ts

```diff
@@ -351,6 +425,27 @@ export class LoginStrategy extends BaseStrategy {
     );
   }
 
+  /**
+   * extracts siwe message and signature either from request body or query
+   * @param req
+   * @returns {string} siwe object - message and signature
+   */
+  extractSiwe(req: Request): SiweMessageReqPayload {
+    if (req.body.identity)
+      return {
+        signature:
+          lookup(req.body.identity, 'signature') ||
+          lookup(req.query, 'signature'),
+        message: req.body.message,
```

Comment:
The `req.body.message` is string. It should be JSON parsed before return

Reply:
Actually it is not in JSON, but plain text. We need to parse it line by line https://docs.login.xyz/sign-in-with-ethereum/quickstart-guide/creating-siwe-messages

Reply:
perhaps this can help https://www.npmjs.com/package/properties-file

Reply:
Maybe the best way to do this is to reuse the `SiweMessage` constructor: https://github.com/spruceid/siwe/blob/053f16bf7dce3ff587e00bc4d3754a29ee9c26e6/packages/siwe/lib/client.ts#L62
```suggestion
        message: new SiweMessage(req.body.message),
```